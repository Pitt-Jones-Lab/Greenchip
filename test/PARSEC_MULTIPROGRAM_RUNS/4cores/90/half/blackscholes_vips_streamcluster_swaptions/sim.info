{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/blackscholes_vips_streamcluster_swaptions',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_90_half_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/blackscholes_vips_streamcluster_swaptions',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpgu6WFg/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb15',
 'pin_version': 'unknown',
 'rusage': (17745.350296000001,
            655.58133599999996,
            192412,
            0,
            0,
            0,
            46874,
            5,
            0,
            11784,
            1176,
            0,
            0,
            0,
            81284894,
            8284106),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/blackscholes_vips_streamcluster_swaptions --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem90.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_90_half_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpgu6WFg/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MULTIPROGRAM_SHELL/4cores/90/half/blackscholes_vips_streamcluster_swaptions/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 7704.9994928836823,
 't_start': 1464035035.19068,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1629396992L}
