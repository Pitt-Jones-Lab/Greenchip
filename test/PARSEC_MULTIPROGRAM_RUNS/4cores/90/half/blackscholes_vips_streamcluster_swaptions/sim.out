                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2387406091 |  2459784435 |  2717595410 |  2433874133
  Cycles                           |  1522619112 |  1522619112 |  1522619112 |  1522619112
  IPC                              |        1.57 |        1.62 |        1.78 |        1.60
  Time (ns)                        |   572413200 |   572413200 |   572413200 |   572413200
  Idle time (ns)                   |      124352 |      555533 |      107421 |      123579
  Idle time (%)                    |        0.0% |        0.1% |        0.0% |        0.0%
Branch predictor stats             |             |             |             |            
  num correct                      |   243017674 |   236411015 |   193337672 |   235030590
  num incorrect                    |     6496876 |     4450780 |     2915033 |     4699630
  misprediction rate               |       2.60% |       1.85% |       1.49% |       1.96%
  mpki                             |        2.72 |        1.81 |        1.07 |        1.93
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   325270862 |   327949252 |   392158107 |   324487362
    num misses                     |        6530 |        2698 |        2176 |        2115
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   792816905 |   806821396 |  1028583008 |   804205303
    num misses                     |      150613 |      991436 |      127954 |      146842
    miss rate                      |       0.02% |       0.12% |       0.01% |       0.02%
    mpki                           |        0.06 |        0.40 |        0.05 |        0.06
  L2 TLB                           |             |             |             |            
    num accesses                   |      157143 |      994134 |      130130 |      148957
    num misses                     |       46530 |       63218 |       60038 |       49443
    miss rate                      |      29.61% |       6.36% |      46.14% |      33.19%
    mpki                           |        0.02 |        0.03 |        0.02 |        0.02
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   325270862 |   327949252 |   392158107 |   324487362
    num cache misses               |      435137 |      405613 |      279388 |      388412
    miss rate                      |       0.13% |       0.12% |       0.07% |       0.12%
    mpki                           |        0.18 |        0.16 |        0.10 |        0.16
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   792816905 |   806821396 |  1028583008 |   804205303
    num cache misses               |     8238674 |     9614900 |     7116584 |     8960750
    miss rate                      |       1.04% |       1.19% |       0.69% |       1.11%
    mpki                           |        3.45 |        3.91 |        2.62 |        3.68
  Cache L2                         |             |             |             |            
    num cache accesses             |     9514802 |    11023335 |     9118192 |    10308126
    num cache misses               |     2607826 |     3721325 |     3275275 |     2816037
    miss rate                      |      27.41% |      33.76% |      35.92% |      27.32%
    mpki                           |        1.09 |        1.51 |        1.21 |        1.16
DRAM summary                       |             |             |             |            
  num dram accesses                |    22125560 |           0 |           0 |           0
  average dram access latency (ns) |      100.05 |         inf |         inf |         inf
  average dram queueing delay      |      100.05 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     2021897 |     2360201 |     2622825 |     2278296
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
